### ISSUE TITLE
Reproduction issues

### ISSUE THREAD

**Issue by xiewende** [xiaofeng94/GMFlowNet#6]
Nice Work！！
Thank you very much for your work! However, when I reproduce your code, I never reach the metric of your paper, is there any training trick ?

Looking forward to your reply！

**Comment by xiaofeng94** [comment]
Hey, can you share more details about the results and the setting? 
BTW, you need to use the newest code to reproduce the results. I used a smaller learning rate by mistake before.

**Comment by xiewende** [comment]
I re-pulled your code, all settings have not changed. I am running on two A100 GPUs, but the result is still not ideal.

**Comment by xiaofeng94** [comment]
Hey, can you reproduce the results with released weights? What's your environment? I'm wondering if you got different environments, like pytorch, cuda. New version of pytorch may change some default settings. For example  `align_corners` in `F.grid_sample()` will be changed from True to False. Those will affect the results a lot.

Another way to check your environment is to reproduce RAFT first. You may train RAFT with my code as follows, 
`python -u train.py --model raft--name raft-chairs --stage chairs --validation chairs --gpus 0 1 --num_steps 120000 --batch_size 10 --lr 0.0004 --image_size 368 496 --wdecay 0.0001`

Also, it's better for you to post more details about your experiments, like what results you got, which training script did you use.

**Comment by xiewende** [comment]
Thanks for your reply！
I checked the problem you mentioned and ran it a few more times, now it meets the metrics of the paper on the Sintel dataset, but still performs a bit worse on Kitti.
In addition, I use the train_gmflownet.sh to run.

**Comment by xiaofeng94** [comment]
Hey, we also found the training was not quite stable on KITTI possibly because there are less data. Also, it seems HD1K data causes the instability. You may remove it but expect lower performance.
